live/app/initializers/router.js

```diff
@@ -0,0 +1,9 @@
+export function initialize(application) {
+  application.inject('route', 'router', 'router:main');
```

Comment:
oui ! très pratique pour savoir dans quelle route on est. Voir http://stackoverflow.com/questions/18302463/get-current-route-name-in-ember